build.sbt

```diff
@@ -44,25 +44,23 @@ inThisBuild(
     // https://www.scala-sbt.org/1.x/docs/Cached-Resolution.html
     updateOptions := updateOptions.value.withCachedResolution(true),
     resolvers ++= Resolver.sonatypeOssRepos("releases"), // libraries that haven't yet synced to maven central
+    assembly / assemblyMergeStrategy := {
```

Comment:
this moves the merge strategy to "ThisBuild" which applies to all projects, so as long as no other project overrides it, it will automatically get used.  This saves faff trying to get it into all the right places.
The reason was because some mime.types file and netty file was conflicting in some places